docs/core/allocation.qmd

```diff
@@ -226,37 +225,6 @@ $$ {#eq-fractionalflowconstraint}
 
 - Flow sign: Furthermore there are the non-negativity constraints for the flows and allocations, see [The optimization variables](allocation.qmd#the-optimization-variables).
 
-## Final notes on the allocation problem
-
-### UserDemands using their own return flow
-
-If not explicitly avoided, UserDemands can use their own return flow in this allocation problem formulation.
-Therefore, return flow of UserDemand is only taken into account by allocation if that return flow is downstream of the UserDemand where it comes from. That is, if there is no path in the directed allocation network from the UserDemand outflow node back to the UserDemand.
-
-# Solving the allocation problem {#sec-solving-allocation}
```

Comment:
I removed this section as it is tedious to maintain and probably doesn't help anyone.